huaweicloud/provider.go

```diff
@@ -608,6 +609,7 @@ func Provider() *schema.Provider {
 			"huaweicloud_waf_instance_group":              waf.ResourceWafInstanceGroup(),
 			"huaweicloud_waf_instance_group_associate":    waf.ResourceWafInstGroupAssociate(),
 			"huaweicloud_waf_reference_table":             waf.ResourceWafReferenceTableV1(),
+			"huaweicloud_csms_secret":                     dew.ResourceCsmsSecret(),
```

Comment:
please move this above, we make this list in alphabetical order.

Reply:
fixed